File: stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/ui/guides/AppStartupInitializers.java
package io.getstream.chat.docs.java.ui.guides;

import android.content.Context;

import androidx.startup.AppInitializer;

import io.getstream.chat.android.ui.initializer.ChatUIInitializer;

/**
 * [App Startup Initializers](https://getstream.io/chat/docs/sdk/android/ui/guides/app-startup-initializers/)
 */
public class AppStartupInitializers {

    public void manualInitialization(Context appContext) {
        AppInitializer.getInstance(appContext).initializeComponent(ChatUIInitializer.class);
    }
}

File: stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/ui/guides/AppStartupInitializers.kt
package io.getstream.chat.docs.kotlin.ui.guides

import android.content.Context
import androidx.startup.AppInitializer
import io.getstream.chat.android.ui.initializer.ChatUIInitializer

/**
 * [App Startup Initializers](https://getstream.io/chat/docs/sdk/android/ui/guides/app-startup-initializers/)
 */
class AppStartupInitializers {

    fun manualInitialization(appContext: Context) {
        AppInitializer.getInstance(appContext).initializeComponent(ChatUIInitializer::class.java)
    }
}
